core-java-modules/core-java-streams-3/src/main/java/com/baeldung/streams/bigdecimals/StreamAddNumbers.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.streams.bigdecimals;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+public class StreamAddNumbers {
+
+    public int addInts() {
```

Comment:
Illustrating these functions with unit tests would be preferable. This means you probably don't want to use `Random`. But you can add up the numbers 0, 1 & 2 from `IntStream.range`

Reply:
I moved the code to a unit test class